src/helpers/userHelper.js

```diff
@@ -13,6 +13,7 @@ const emailSender = require("../utilities/emailSender");
 const logger = require("../startup/logger");
 const hasPermission = require("../utilities/permissions");
 const Reason = require("../models/reason");
+const sendMail = require("../utilities/nodemailer")
```

Comment:
Hi @mounica-dingari-by 
I am getting Error: Cannot find module '../utilities/nodemailer'. I checked the utilities folder, but there is no nodemailer file in that.
<img width="573" alt="Screenshot 2023-10-28 at 11 43 27" src="https://github.com/OneCommunityGlobal/HGNRest/assets/94716419/c44050d3-f547-4b5b-b318-a58bd53c7cd1">